.eslintrc

```diff
@@ -216,12 +210,23 @@
     },
     {
       "files": ["*.pw.ts*", "*.test-pw.ts*", "playwright/**/*.ts*"],
+      "plugins": ["playwright"],
       "rules": {
+        "playwright/no-commented-out-tests": "error",
+        "playwright/no-focused-test": "error",
+        "playwright/no-skipped-test": "warn",
         "no-return-await": "warn",
         "no-return-assign": "warn",
         "no-await-in-loop": "warn",
         "@typescript-eslint/no-floating-promises": "error"
+      },
+      "settings": {
+        "playwright": {
+          "messages": {
+            "noSkippedTest": "Skipped test. If due to an issue that cannot be resolved currently, please raise a Github issue or ticket and reference it in a comment."
```

Comment:
Just a nit pick really, but should this state "please raise a Github issue or **_Jira_** ticket"?